Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Ride.hs

```diff
@@ -267,7 +267,7 @@ buildRide driver booking ghrId otp enableFrequentLocationUpdates clientId previo
         estimatedEndTimeRange = Nothing,
         rideTags = Nothing,
         hasStops = booking.hasStops,
-        isPickupOrDestinationEdited = Nothing
+        isPickupOrDestinationEdited = Just False
```

Comment:
What is this change ?

Reply:
Incase null case is not handled in json logic for any Maybe field, jsonlogic gives null
That's why added just for precaution